frontend/pages/ManageControlsPage/MacOSSettings/cards/CustomSettings/CustomSettings.tsx

```diff
@@ -77,18 +83,27 @@ const CustomSettings = ({
     setShowDeleteProfileModal(false);
   };
 
-  const onDeleteProfile = async (profileId: number) => {
+  const onDeleteProfile = async ({
+    profileId,
+    profileName,
+  }: IDeleteProfileProps) => {
     try {
-      await mdmAPI.deleteProfile(profileId);
-      onProfileDelete();
-      renderFlash("success", "Successfully deleted!");
+      profileName === "Disk encryption"
+        ? mdmAPI.updateAppleMdmSettings(false, currentTeam?.id || 0)
+        : mdmAPI.deleteProfile(profileId);
+      const timer = setTimeout(() => {
+        renderFlash("success", "Successfully deleted!");
+        refetchProfiles();
+        refetchConfig();
+      }, 1000);
```

Comment:
It takes a second, I don't know why, for the profiles and config to be updated after using mdmAPI endpoint. I put a 1 second wait and it works, but it doesn't work without it.

Reply:
Reused `mdmAPI.updateAppleMdmSettings(false, currentTeam?.id || 0)` from the disk encryption page because the functionality should be the same deleting the disk encryption profile and turning it off, correct?